src/spaces/prophoto.js

```diff
@@ -1,43 +1,50 @@
 import Color from "./srgb.js";
 
+/**
+ * Reference Output Medium Metric (ROMM) RGB was developed by Kodak as ProPhoto RGB
+ * It has been standardized as Part 2 of ISO 22028
+ * There is also a closely related Reference _Input_ Medium Metric (RIMM) RGB as Part 3.
+ * 
+ * @see https://www.iso.org/standard/56591.html ISO 22028-2
+ * @see https://webstore.ansi.org/Standards/I3A/ANSII3AIT1076662002 ANSI/I3A IT10.7666-2002
+ * @see http://www.color.org/ROMMRGB.pdf
+ */
 Color.defineSpace({
 	inherits: "srgb",
 	id: "prophoto",
 	name: "ProPhoto",
 	cssId: "prophoto-rgb",
 	white: Color.whites.D50,
+	α /*= a + 1*/: 1.0,
```

Comment:
Where do these constants come from? Did you buy the standards?

Reply:
My alma mater has full ISO and DIN subscriptions, but I should check again since I wrote this part a while ago.